rasa/shared/importers/rasa.py

```diff
@@ -45,7 +49,7 @@ async def get_stories(
         use_e2e: bool = False,
         exclusion_percentage: Optional[int] = None,
     ) -> StoryGraph:
-
+        """Retrieves training stories / rules (see parent class for full docstring)."""
```

Comment:
Why does this implementation not branch on `use_e2e` like the one in `MultiProjectImporter`?

Reply:
don't we need to use `_conversation_test_files` path?

Reply:
We could do so. I found it a bit risky / fragile to select different training data based on a flag. That's why I preferred introducing a separate method on the class.

Reply:
I guess my question then is why do we have:
```
if use_e2e:
            return await self.get_conversation_tests()
```
in `MultiProjectImporter.get_stories`?

Reply:
because the multiproject importer included test stories during training: https://github.com/RasaHQ/rasa/issues/5913. This couldn't really happen in the normal importer because users just provided different paths were one directory couldn't contain both. 

Ideally we'd explicitly exclude test files from `get_stories` as well in `RasaFileImporter` 🤔

Reply:
🤯 I'm not sure I understand. We have `get_stories` _and_ `get_conversation_tests` now. Can't we just call the correct method where needed? Why do we need to sometimes return conversation tests from the `get_stories` method in some importers?

Reply:
true 👍🏻 Will remove the implementation from the multiprojectimporter